doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1163,6 +1163,8 @@ Documentation
   * The Device configuration guides section and moved its contents to :ref:`ug_app_dev`.
   * The Advanced building procedures page and moved its contents to the :ref:`building` page.
   * nRF70 Series support is upstreamed to Zephyr, hence the documentation is removed from the |NCS|.
+  * The standalone pages for getting started with nRF52 Series and with the nRF5340 DK.
+    These pages have been replaced by the `Quick Start`_ app, which now supports the nRF52 Series devices and the nRF5340 DK.
```

Comment:
```suggestion
    These pages have been replaced by the ref:`gsg_guides` page, with details of `Quick Start`_ app that now supports the nRF52 Series devices and the nRF5340 DK.
```

Reply:
The phrasing is intentional to point the users to the app and skip the unnecessary pages.